In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.5.0


In [3]:
train_df = pd.read_csv("data/titanic/train.csv")
serving_df = pd.read_csv("data/titanic/test.csv")

train_df.head(10)


PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

Prepare dataset
We will apply the following transformations on the dataset.

Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [4]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

In [5]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


### Convert Pandas dataset to TensorFlow Dataset

In [6]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

Metal device set to: Apple M1 Pro


In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 23-08-06 19:44:32.0942 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:44:32.0952 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:44:32.0952 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:44:35.2200 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpeepd9k9e/model/ with prefix d15ff54cd02f457a
[INFO 23-08-06 19:44:35.2370 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:44:35.2370 IST kernel.cc:1075] Use fast generic engine


Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: could not get source code
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: could not get source code
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Accuracy: 0.804347813129425 Loss:0.8922085762023926


## Train model with improved default parameters
Now you'll use some specific parameters when creating the GBT model



In [8]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 23-08-06 19:46:01.7311 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:46:01.7311 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:46:01.7311 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.77173912525177 Loss:1.0344746112823486


[INFO 23-08-06 19:46:02.1821 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmps0d6sbri/model/ with prefix a1b765f3c5e645d1
[INFO 23-08-06 19:46:02.1948 IST decision_forest.cc:660] Model loaded with 34 root(s), 1782 node(s), and 10 input feature(s).
[INFO 23-08-06 19:46:02.1949 IST abstract_model.cc:1312] Engine "GradientBoostedTreesGeneric" built
[INFO 23-08-06 19:46:02.1949 IST kernel.cc:1075] Use fast generic engine


In [9]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.542176 ################
    2.           "Age"  0.362914 ########
    3.          "Fare"  0.267978 ####
    4.          "Name"  0.210150 #
    5.        "Pclass"  0.182221 
    6. "Ticket_number"  0.182105 
    7.      "Embarked"  0.177187 
    8.   "Ticket_item"  0.176830 
    9.         "Parch"  0.176170 
   10.         "SibSp"  0.171814 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 28.000000 ####

### Make predictions

In [11]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="data/titanic/submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)
!head data/titanic/submission.csv

Submission exported to data/titanic/submission.csv
PassengerId,Survived
892,0
893,0
894,0
895,0
896,0
897,0
898,0
899,0
900,1


### Training a model with hyperparameter tunning
Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).

In [12]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxftlpo5p as temporary training directory


[WARNING 23-08-06 19:49:56.0870 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:49:56.0871 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:49:56.0871 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 23-08-06 19:50:26.7430 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxftlpo5p/model/ with prefix e91a0cc7d3f844ed
[INFO 23-08-06 19:50:26.7590 IST decision_forest.cc:660] Model loaded with 35 root(s), 1085 node(s), and 12 input feature(s).
[INFO 23-08-06 19:50:26.7590 IST abstract_model.cc:1312] Engine "GradientBoostedTreesGeneric" built
[INFO 23-08-06 19:50:26.7590 IST kernel.cc:1075] Use fast generic engine


Accuracy: 0.8767123222351074 Loss:0.6833678483963013


### Making an ensemble
Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the honest parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [14]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

i:0


[WARNING 23-08-06 19:52:01.0157 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:01.0157 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:01.0157 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 23-08-06 19:52:01.3424 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpwv__p1bj/model/ with prefix ad09599f7af44868
[INFO 23-08-06 19:52:01.3511 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:52:01.3512 IST kernel.cc:1075] Use fast generic engine


i:1


[WARNING 23-08-06 19:52:01.5978 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:01.5978 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:01.5978 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:02.0050 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpzc4r31ox/model/ with prefix 34f5f4e484a24d0b
[INFO 23-08-06 19:52:02.0665 IST kernel.cc:1075] Use fast generic engine


i:2


[WARNING 23-08-06 19:52:02.3499 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:02.3499 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:02.3499 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 23-08-06 19:52:02.7945 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp4j51qd8o/model/ with prefix 88e1669ca16c4bb1
[INFO 23-08-06 19:52:02.8062 IST kernel.cc:1075] Use fast generic engine


i:3


[WARNING 23-08-06 19:52:03.0549 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:03.0549 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:03.0549 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:03.4187 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpq6p64x5w/model/ with prefix 4e0034b8b3e940b8
[INFO 23-08-06 19:52:03.4298 IST kernel.cc:1075] Use fast generic engine


i:4


[WARNING 23-08-06 19:52:03.6765 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:03.6765 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:03.6765 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:03.9767 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpw5z5fot3/model/ with prefix b65c8691f4314ee7
[INFO 23-08-06 19:52:03.9886 IST kernel.cc:1075] Use fast generic engine


i:5


[WARNING 23-08-06 19:52:04.2391 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:04.2391 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:04.2391 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:04.5578 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpx9wxj1x2/model/ with prefix 8b0500b731b14522
[INFO 23-08-06 19:52:04.5688 IST kernel.cc:1075] Use fast generic engine


i:6


[WARNING 23-08-06 19:52:04.8138 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:04.8138 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:04.8138 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:05.2050 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpo1js43p7/model/ with prefix 1169f8db4387447a
[INFO 23-08-06 19:52:05.2163 IST kernel.cc:1075] Use fast generic engine


i:7


[WARNING 23-08-06 19:52:05.4621 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:05.4621 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:05.4621 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:05.7963 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpqfof6d2i/model/ with prefix 859b30cd6e824dba
[INFO 23-08-06 19:52:05.8120 IST kernel.cc:1075] Use fast generic engine


i:8


[WARNING 23-08-06 19:52:06.0556 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:06.0557 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:06.0557 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:06.4991 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpfnd3ft_s/model/ with prefix 348c2bdfd2ab4100
[INFO 23-08-06 19:52:06.5161 IST kernel.cc:1075] Use fast generic engine


i:9


[WARNING 23-08-06 19:52:07.0664 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:07.0664 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:07.0664 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:07.5223 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp30l8fr7w/model/ with prefix aa3dbd55a8d44a8e
[INFO 23-08-06 19:52:07.5389 IST kernel.cc:1075] Use fast generic engine


i:10


[WARNING 23-08-06 19:52:07.7850 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:07.7850 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:07.7851 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:08.1224 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpbg9z3o5u/model/ with prefix c0bad26475be4075
[INFO 23-08-06 19:52:08.1373 IST kernel.cc:1075] Use fast generic engine


i:11


[WARNING 23-08-06 19:52:08.3847 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:08.3847 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:08.3847 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:08.8494 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpahypkg5b/model/ with prefix f92d256431854cfe
[INFO 23-08-06 19:52:08.8704 IST kernel.cc:1075] Use fast generic engine


i:12


[WARNING 23-08-06 19:52:09.1176 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:09.1177 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:09.1177 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:09.4969 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp_026ob2_/model/ with prefix c6fbfc1499b04801
[INFO 23-08-06 19:52:09.5131 IST kernel.cc:1075] Use fast generic engine


i:13


[WARNING 23-08-06 19:52:09.7656 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:09.7656 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:09.7656 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:10.4612 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpka7b1flp/model/ with prefix 89139ef6c53645c2
[INFO 23-08-06 19:52:10.4775 IST kernel.cc:1075] Use fast generic engine


i:14


[WARNING 23-08-06 19:52:10.7261 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:10.7261 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:10.7261 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:11.0830 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp26csj1_z/model/ with prefix 44cf9becada64840
[INFO 23-08-06 19:52:11.0964 IST kernel.cc:1075] Use fast generic engine


i:15


[WARNING 23-08-06 19:52:11.3434 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:11.3434 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:11.3434 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:11.6688 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp_2g8b4fr/model/ with prefix adb85438de244095
[INFO 23-08-06 19:52:11.6844 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:52:11.6844 IST kernel.cc:1075] Use fast generic engine


i:16


[WARNING 23-08-06 19:52:11.9280 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:11.9280 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:11.9280 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:12.3335 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpf1mj9n_k/model/ with prefix 26ad5ddb71354bd1
[INFO 23-08-06 19:52:12.3481 IST kernel.cc:1075] Use fast generic engine


i:17


[WARNING 23-08-06 19:52:12.5944 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:12.5944 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:12.5944 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:12.9239 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp5srurgvo/model/ with prefix b35bd4a830c3413f
[INFO 23-08-06 19:52:12.9371 IST kernel.cc:1075] Use fast generic engine


i:18


[WARNING 23-08-06 19:52:13.1812 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:13.1812 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:13.1812 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:13.6209 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmppj24e0uj/model/ with prefix 86de52d4603e47c1
[INFO 23-08-06 19:52:13.6300 IST kernel.cc:1075] Use fast generic engine


i:19


[WARNING 23-08-06 19:52:13.8747 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:13.8748 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:13.8748 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:14.2686 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp5nhvgb0k/model/ with prefix 6420a22bdbb14229
[INFO 23-08-06 19:52:14.2800 IST kernel.cc:1075] Use fast generic engine


i:20


[WARNING 23-08-06 19:52:14.5282 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:14.5282 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:14.5282 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:14.8644 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpoko2vsdg/model/ with prefix 66c8e1673fbe421b
[INFO 23-08-06 19:52:14.8756 IST kernel.cc:1075] Use fast generic engine


i:21


[WARNING 23-08-06 19:52:15.1279 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:15.1280 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:15.1280 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:15.4765 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpvt70jr1v/model/ with prefix 8a92e62548414862
[INFO 23-08-06 19:52:15.4945 IST kernel.cc:1075] Use fast generic engine


i:22


[WARNING 23-08-06 19:52:15.7489 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:15.7489 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:15.7489 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:16.1372 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpsivz5es4/model/ with prefix 24366978fea841b1
[INFO 23-08-06 19:52:16.1551 IST kernel.cc:1075] Use fast generic engine


i:23


[WARNING 23-08-06 19:52:16.4216 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:16.4216 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:16.4216 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:16.7738 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp_fsy0jme/model/ with prefix a15eed1e840348c5
[INFO 23-08-06 19:52:16.7820 IST kernel.cc:1075] Use fast generic engine


i:24


[WARNING 23-08-06 19:52:17.0516 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:17.0517 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:17.0517 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:17.3978 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp8z6h903x/model/ with prefix ae3dc32c584545f1
[INFO 23-08-06 19:52:17.4151 IST kernel.cc:1075] Use fast generic engine


i:25


[WARNING 23-08-06 19:52:17.6745 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:17.6745 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:17.6745 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:18.4931 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp7s__izem/model/ with prefix 8a2bde14c2774d68
[INFO 23-08-06 19:52:18.5154 IST kernel.cc:1075] Use fast generic engine


i:26


[WARNING 23-08-06 19:52:18.7715 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:18.7715 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:18.7715 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:19.0979 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp2phcti5z/model/ with prefix 5722ea7533eb4f34
[INFO 23-08-06 19:52:19.1166 IST kernel.cc:1075] Use fast generic engine


i:27


[WARNING 23-08-06 19:52:19.3913 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:19.3913 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:19.3913 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:19.8213 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp07i8jf9t/model/ with prefix 4fe68a0cdc934a1b
[INFO 23-08-06 19:52:19.8340 IST kernel.cc:1075] Use fast generic engine


i:28


[WARNING 23-08-06 19:52:20.0861 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:20.0861 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:20.0861 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:20.3844 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpeeypjps0/model/ with prefix 0401c371c2c44a97
[INFO 23-08-06 19:52:20.3943 IST kernel.cc:1075] Use fast generic engine


i:29


[WARNING 23-08-06 19:52:20.6625 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:20.6625 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:20.6625 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:21.0152 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpewtczkq0/model/ with prefix ef796367291f4e3a
[INFO 23-08-06 19:52:21.0221 IST kernel.cc:1075] Use fast generic engine


i:30


[WARNING 23-08-06 19:52:21.2783 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:21.2783 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:21.2783 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:21.6895 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpq6wczf99/model/ with prefix b5a5be2ef0264cda
[INFO 23-08-06 19:52:21.7086 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:52:21.7086 IST kernel.cc:1075] Use fast generic engine


i:31


[WARNING 23-08-06 19:52:21.9596 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:21.9596 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:21.9596 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:22.4537 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpz676yoxv/model/ with prefix 1a60f18aee3c4016
[INFO 23-08-06 19:52:22.4763 IST kernel.cc:1075] Use fast generic engine


i:32


[WARNING 23-08-06 19:52:22.7470 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:22.7471 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:22.7471 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:23.2191 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpr0r__01a/model/ with prefix 994972e6f2fb4ae1
[INFO 23-08-06 19:52:23.2375 IST kernel.cc:1075] Use fast generic engine


i:33


[WARNING 23-08-06 19:52:23.4965 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:23.4965 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:23.4965 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:23.8790 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpfa9tzwat/model/ with prefix f9c77b1b74804889
[INFO 23-08-06 19:52:23.8911 IST kernel.cc:1075] Use fast generic engine


i:34


[WARNING 23-08-06 19:52:24.1423 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:24.1423 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:24.1423 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:24.4946 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmph7013z45/model/ with prefix 9e0b0a9bde6a4eb3
[INFO 23-08-06 19:52:24.5078 IST kernel.cc:1075] Use fast generic engine


i:35


[WARNING 23-08-06 19:52:24.7718 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:24.7718 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:24.7718 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:25.1835 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpyw7npw3h/model/ with prefix 11c4b758c6f14289
[INFO 23-08-06 19:52:25.1907 IST kernel.cc:1075] Use fast generic engine


i:36


[WARNING 23-08-06 19:52:25.4585 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:25.4585 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:25.4585 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:25.7970 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxf71jwi_/model/ with prefix 872f47718bf54942
[INFO 23-08-06 19:52:25.8030 IST kernel.cc:1075] Use fast generic engine


i:37


[WARNING 23-08-06 19:52:26.0573 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:26.0574 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:26.0574 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:26.3987 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpv19y3uvw/model/ with prefix ee8598dc7961431f
[INFO 23-08-06 19:52:26.4148 IST kernel.cc:1075] Use fast generic engine


i:38


[WARNING 23-08-06 19:52:26.6833 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:26.6833 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:26.6833 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:27.0972 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpkgznayrm/model/ with prefix e18728fe2791406d
[INFO 23-08-06 19:52:27.1160 IST kernel.cc:1075] Use fast generic engine


i:39


[WARNING 23-08-06 19:52:27.4056 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:27.4056 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:27.4056 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:27.8403 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpmryejrph/model/ with prefix 66725ffebc494b1e
[INFO 23-08-06 19:52:27.8585 IST kernel.cc:1075] Use fast generic engine


i:40


[WARNING 23-08-06 19:52:28.1151 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:28.1151 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:28.1151 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:28.4798 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp_tjuj92v/model/ with prefix fc26fa4b405e4737
[INFO 23-08-06 19:52:28.4972 IST kernel.cc:1075] Use fast generic engine


i:41


[WARNING 23-08-06 19:52:28.7419 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:28.7419 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:28.7419 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:29.1582 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp1uojp80g/model/ with prefix 26648907be084b39
[INFO 23-08-06 19:52:29.1728 IST kernel.cc:1075] Use fast generic engine


i:42


[WARNING 23-08-06 19:52:29.4227 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:29.4228 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:29.4228 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:29.7874 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpbf8_iwvk/model/ with prefix e7511ef9bf694003
[INFO 23-08-06 19:52:29.7946 IST kernel.cc:1075] Use fast generic engine


i:43


[WARNING 23-08-06 19:52:30.0480 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:30.0480 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:30.0480 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:30.4558 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpwvzj2u32/model/ with prefix e37330e9802e4026
[INFO 23-08-06 19:52:30.4707 IST kernel.cc:1075] Use fast generic engine


i:44


[WARNING 23-08-06 19:52:30.7238 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:30.7238 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:30.7238 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:31.0708 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpq0dz_ngy/model/ with prefix f4255b14a6224dac
[INFO 23-08-06 19:52:31.0840 IST kernel.cc:1075] Use fast generic engine


i:45


[WARNING 23-08-06 19:52:31.8571 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:31.8571 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:31.8572 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:32.1670 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpz5ujzjzk/model/ with prefix a415850e49ed4356
[INFO 23-08-06 19:52:32.1730 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:52:32.1730 IST kernel.cc:1075] Use fast generic engine


i:46


[WARNING 23-08-06 19:52:32.4451 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:32.4451 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:32.4451 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:32.8602 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpt4uijvku/model/ with prefix b19cb00fcec14f2b
[INFO 23-08-06 19:52:32.8681 IST kernel.cc:1075] Use fast generic engine


i:47


[WARNING 23-08-06 19:52:33.1292 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:33.1292 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:33.1293 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:33.4802 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpfevc2rio/model/ with prefix de74afa34d434b1a
[INFO 23-08-06 19:52:33.4910 IST kernel.cc:1075] Use fast generic engine


i:48


[WARNING 23-08-06 19:52:33.7614 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:33.7615 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:33.7615 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:34.1143 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpp8bz0_6e/model/ with prefix f779f0723e354cd6
[INFO 23-08-06 19:52:34.1301 IST kernel.cc:1075] Use fast generic engine


i:49


[WARNING 23-08-06 19:52:34.4104 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:34.4104 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:34.4104 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:34.7984 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpbn41hyv3/model/ with prefix 5f533b13d8d24d39
[INFO 23-08-06 19:52:34.8115 IST kernel.cc:1075] Use fast generic engine


i:50


[WARNING 23-08-06 19:52:35.1002 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:35.1002 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:35.1002 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:35.5933 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp919m7vj2/model/ with prefix 33ffd329fa264d5b
[INFO 23-08-06 19:52:35.6109 IST kernel.cc:1075] Use fast generic engine


i:51


[WARNING 23-08-06 19:52:35.8722 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:35.8722 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:35.8722 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:36.3345 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpuwnb9peb/model/ with prefix 8c97bc051b8944ed
[INFO 23-08-06 19:52:36.3532 IST kernel.cc:1075] Use fast generic engine


i:52


[WARNING 23-08-06 19:52:36.6004 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:36.6005 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:36.6005 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:36.9516 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpf2302leb/model/ with prefix 01d696e299d84fcb
[INFO 23-08-06 19:52:36.9673 IST kernel.cc:1075] Use fast generic engine


i:53


[WARNING 23-08-06 19:52:37.2127 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:37.2127 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:37.2128 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:37.5482 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxyvgv6yi/model/ with prefix 93c9e3ad666440e3
[INFO 23-08-06 19:52:37.5594 IST kernel.cc:1075] Use fast generic engine


i:54


[WARNING 23-08-06 19:52:37.8021 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:37.8021 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:37.8021 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:38.1510 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpmz4gdfid/model/ with prefix d7bcc0def8884ed7
[INFO 23-08-06 19:52:38.1629 IST kernel.cc:1075] Use fast generic engine


i:55


[WARNING 23-08-06 19:52:38.4081 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:38.4081 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:38.4082 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:38.8212 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpzvxpi5wn/model/ with prefix 995138011b1e48f4
[INFO 23-08-06 19:52:38.8408 IST kernel.cc:1075] Use fast generic engine


i:56


[WARNING 23-08-06 19:52:39.0892 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:39.0892 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:39.0893 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:39.4337 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp_ld9fo09/model/ with prefix b8a443c1559a46e9
[INFO 23-08-06 19:52:39.4505 IST kernel.cc:1075] Use fast generic engine


i:57


[WARNING 23-08-06 19:52:39.6951 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:39.6952 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:39.6952 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:40.0459 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpu29cuzvh/model/ with prefix 81fa7673f8c740bc
[INFO 23-08-06 19:52:40.0558 IST kernel.cc:1075] Use fast generic engine


i:58


[WARNING 23-08-06 19:52:40.3022 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:40.3023 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:40.3023 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:40.7860 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmptsirm4w1/model/ with prefix 3af0b8c605844459
[INFO 23-08-06 19:52:40.8073 IST kernel.cc:1075] Use fast generic engine


i:59


[WARNING 23-08-06 19:52:41.0556 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:41.0556 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:41.0556 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:41.5187 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxspf5gsh/model/ with prefix f737de732dec4f7e
[INFO 23-08-06 19:52:41.5376 IST kernel.cc:1075] Use fast generic engine


i:60


[WARNING 23-08-06 19:52:41.7870 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:41.7870 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:41.7870 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:42.0908 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp1nd20lh_/model/ with prefix ac91f012e5f749d2
[INFO 23-08-06 19:52:42.1011 IST kernel.cc:1075] Use fast generic engine


i:61


[WARNING 23-08-06 19:52:42.3483 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:42.3483 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:42.3483 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:42.6985 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpp17adk5u/model/ with prefix 5e39152bf3034724
[INFO 23-08-06 19:52:42.7098 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:52:42.7098 IST kernel.cc:1075] Use fast generic engine


i:62


[WARNING 23-08-06 19:52:42.9548 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:42.9548 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:42.9548 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:43.3369 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpzlj_wd0t/model/ with prefix 2a5d7b2f6bcf4c0d
[INFO 23-08-06 19:52:43.3499 IST kernel.cc:1075] Use fast generic engine


i:63


[WARNING 23-08-06 19:52:43.5970 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:43.5971 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:43.5971 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:43.9312 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpjizxxqyr/model/ with prefix fb09386faafb4126
[INFO 23-08-06 19:52:43.9465 IST kernel.cc:1075] Use fast generic engine


i:64


[WARNING 23-08-06 19:52:44.1906 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:44.1906 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:44.1906 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:44.5459 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp2lvdld39/model/ with prefix 775cb1d8fc5d48ad
[INFO 23-08-06 19:52:44.5583 IST kernel.cc:1075] Use fast generic engine


i:65


[WARNING 23-08-06 19:52:45.3625 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:45.3625 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:45.3625 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:45.7646 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmps6fzjmm3/model/ with prefix af093ff533a54f20
[INFO 23-08-06 19:52:45.7827 IST kernel.cc:1075] Use fast generic engine


i:66


[WARNING 23-08-06 19:52:46.0510 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:46.0510 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:46.0510 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:46.3757 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp8gu0k3ln/model/ with prefix 6899701955c9491f
[INFO 23-08-06 19:52:46.3881 IST kernel.cc:1075] Use fast generic engine


i:67


[WARNING 23-08-06 19:52:46.6359 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:46.6359 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:46.6359 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:47.0032 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmps32s4kuq/model/ with prefix 3045138640e54ef1
[INFO 23-08-06 19:52:47.0177 IST kernel.cc:1075] Use fast generic engine


i:68


[WARNING 23-08-06 19:52:47.2646 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:47.2646 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:47.2646 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:47.7743 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp3tklafxb/model/ with prefix acd3e1c723004aba
[INFO 23-08-06 19:52:47.7944 IST kernel.cc:1075] Use fast generic engine


i:69


[WARNING 23-08-06 19:52:48.0433 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:48.0434 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:48.0434 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:48.4518 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpnrk3ze5f/model/ with prefix 2524fcefff6f4cbd
[INFO 23-08-06 19:52:48.4701 IST kernel.cc:1075] Use fast generic engine


i:70


[WARNING 23-08-06 19:52:48.7177 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:48.7178 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:48.7178 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:49.0491 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmph4qr1my1/model/ with prefix 187a3b080b234189
[INFO 23-08-06 19:52:49.0634 IST kernel.cc:1075] Use fast generic engine


i:71


[WARNING 23-08-06 19:52:49.3183 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:49.3183 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:49.3183 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:49.6320 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp378bj_ok/model/ with prefix b246d396555541f2
[INFO 23-08-06 19:52:49.6433 IST kernel.cc:1075] Use fast generic engine


i:72


[WARNING 23-08-06 19:52:49.8891 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:49.8891 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:49.8891 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:50.2579 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpb6fpnv73/model/ with prefix a821a8e2ab284cb9
[INFO 23-08-06 19:52:50.2736 IST kernel.cc:1075] Use fast generic engine


i:73


[WARNING 23-08-06 19:52:50.5233 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:50.5233 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:50.5233 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:50.8302 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp8vxicjil/model/ with prefix b48ab32c157a46ff
[INFO 23-08-06 19:52:50.8424 IST kernel.cc:1075] Use fast generic engine


i:74


[WARNING 23-08-06 19:52:51.0878 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:51.0879 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:51.0879 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:51.5728 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxb4al2jd/model/ with prefix 79f0cab8c6f341ae
[INFO 23-08-06 19:52:51.5875 IST kernel.cc:1075] Use fast generic engine


i:75


[WARNING 23-08-06 19:52:51.8389 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:51.8389 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:51.8389 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:52.1586 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpegpgfbnu/model/ with prefix f00cdea5c5984fce
[INFO 23-08-06 19:52:52.1711 IST kernel.cc:1075] Use fast generic engine


i:76


[WARNING 23-08-06 19:52:52.4174 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:52.4175 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:52.4175 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:52.7437 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmptsu40_7q/model/ with prefix ab5df21f41634fce
[INFO 23-08-06 19:52:52.7539 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:52:52.7539 IST kernel.cc:1075] Use fast generic engine


i:77


[WARNING 23-08-06 19:52:53.0000 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:53.0001 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:53.0001 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:53.3724 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp2_h7f1n9/model/ with prefix 09cd6f0125554ade
[INFO 23-08-06 19:52:53.3880 IST kernel.cc:1075] Use fast generic engine


i:78


[WARNING 23-08-06 19:52:53.6368 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:53.6368 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:53.6368 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:53.9322 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp47wl8bvn/model/ with prefix 9d67321a298c4d8e
[INFO 23-08-06 19:52:53.9429 IST kernel.cc:1075] Use fast generic engine


i:79


[WARNING 23-08-06 19:52:54.1873 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:54.1873 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:54.1873 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:54.6070 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpxaljtvkm/model/ with prefix 7052aecfc1e8454e
[INFO 23-08-06 19:52:54.6236 IST kernel.cc:1075] Use fast generic engine


i:80


[WARNING 23-08-06 19:52:54.9003 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:54.9003 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:54.9003 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:55.2570 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpiuo0hov4/model/ with prefix 102136350da94d88
[INFO 23-08-06 19:52:55.2717 IST kernel.cc:1075] Use fast generic engine


i:81


[WARNING 23-08-06 19:52:55.5217 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:55.5218 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:55.5218 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:55.9172 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmppz6576o6/model/ with prefix fec8b40982e240da
[INFO 23-08-06 19:52:55.9315 IST kernel.cc:1075] Use fast generic engine


i:82


[WARNING 23-08-06 19:52:56.1811 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:56.1811 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:56.1811 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:56.5889 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpidtbv5np/model/ with prefix 2d772b0fe2a24e74
[INFO 23-08-06 19:52:56.6069 IST kernel.cc:1075] Use fast generic engine


i:83


[WARNING 23-08-06 19:52:56.8563 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:56.8564 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:56.8564 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:57.2636 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpg2oksebt/model/ with prefix b3ac2128dda8436d
[INFO 23-08-06 19:52:57.2796 IST kernel.cc:1075] Use fast generic engine


i:84


[WARNING 23-08-06 19:52:57.5255 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:57.5255 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:57.5255 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:57.9046 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp1idjlgnn/model/ with prefix d44becd073c34c0d
[INFO 23-08-06 19:52:57.9253 IST kernel.cc:1075] Use fast generic engine


i:85


[WARNING 23-08-06 19:52:58.1698 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:58.1698 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:58.1698 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:58.5383 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp2wfjujc5/model/ with prefix 2ad4e46002ce4b53
[INFO 23-08-06 19:52:58.5497 IST kernel.cc:1075] Use fast generic engine


i:86


[WARNING 23-08-06 19:52:58.7963 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:58.7963 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:58.7963 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:52:59.2801 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpdmi502e8/model/ with prefix dfce675bc0e347d0
[INFO 23-08-06 19:52:59.2996 IST kernel.cc:1075] Use fast generic engine


i:87


[WARNING 23-08-06 19:52:59.5507 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:59.5507 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:52:59.5507 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:00.4561 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp1yey7l8e/model/ with prefix 65004b8ec4ac4be5
[INFO 23-08-06 19:53:00.4711 IST kernel.cc:1075] Use fast generic engine


i:88


[WARNING 23-08-06 19:53:00.7216 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:00.7216 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:00.7216 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:01.1238 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp9yhck66s/model/ with prefix b77509e88fc745d9
[INFO 23-08-06 19:53:01.1368 IST kernel.cc:1075] Use fast generic engine


i:89


[WARNING 23-08-06 19:53:01.3894 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:01.3894 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:01.3894 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:01.7540 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpf1c74hk9/model/ with prefix c5148c09e6cd4847
[INFO 23-08-06 19:53:01.7636 IST kernel.cc:1075] Use fast generic engine


i:90


[WARNING 23-08-06 19:53:02.0265 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:02.0265 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:02.0265 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:02.3809 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp1dqmnu88/model/ with prefix 5aef0916ebdb4dcc
[INFO 23-08-06 19:53:02.3885 IST kernel.cc:1075] Use fast generic engine


i:91


[WARNING 23-08-06 19:53:02.6374 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:02.6375 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:02.6375 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:03.0523 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpd3ta87mr/model/ with prefix 155f820505e2418c
[INFO 23-08-06 19:53:03.0667 IST abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-08-06 19:53:03.0667 IST kernel.cc:1075] Use fast generic engine


i:92


[WARNING 23-08-06 19:53:03.3177 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:03.3177 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:03.3177 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:03.8732 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp3i13i1lq/model/ with prefix 9e3aeb4ed04f47d8
[INFO 23-08-06 19:53:03.8909 IST kernel.cc:1075] Use fast generic engine


i:93


[WARNING 23-08-06 19:53:04.1728 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:04.1728 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:04.1728 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:04.5074 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpw3430n0q/model/ with prefix f3d0e48b6d1b4c84
[INFO 23-08-06 19:53:04.5176 IST kernel.cc:1075] Use fast generic engine


i:94


[WARNING 23-08-06 19:53:04.7756 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:04.7756 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:04.7756 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:05.1255 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmprgyezp4v/model/ with prefix c7d2d3b418c94be3
[INFO 23-08-06 19:53:05.1383 IST kernel.cc:1075] Use fast generic engine


i:95


[WARNING 23-08-06 19:53:05.3899 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:05.3899 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:05.3900 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:05.7506 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmplxhsf6_b/model/ with prefix de8026a0b70c4e27
[INFO 23-08-06 19:53:05.7635 IST kernel.cc:1075] Use fast generic engine


i:96


[WARNING 23-08-06 19:53:06.0079 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:06.0079 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:06.0079 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:06.3808 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmp3a44ym8d/model/ with prefix c3f82b391e4f4493
[INFO 23-08-06 19:53:06.3974 IST kernel.cc:1075] Use fast generic engine


i:97


[WARNING 23-08-06 19:53:06.6627 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:06.6627 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:06.6627 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:07.0154 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpyk41yaqc/model/ with prefix df661de86ef244cf
[INFO 23-08-06 19:53:07.0316 IST kernel.cc:1075] Use fast generic engine


i:98


[WARNING 23-08-06 19:53:07.3000 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:07.3000 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:07.3000 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:07.8891 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpfv_iqsc7/model/ with prefix 9e67fb4247c44d9e
[INFO 23-08-06 19:53:07.9178 IST kernel.cc:1075] Use fast generic engine


i:99


[WARNING 23-08-06 19:53:08.1704 IST gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:08.1704 IST gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-08-06 19:53:08.1704 IST gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-08-06 19:53:08.8761 IST kernel.cc:1243] Loading model from path /var/folders/7p/nhr_8mz933v_4r1mwyd3_jg40000gp/T/tmpx6n4jn0u/model/ with prefix a8453c8b43a84bfc
[INFO 23-08-06 19:53:08.9089 IST kernel.cc:1075] Use fast generic engine


Submission exported to data/titanic/submission.csv
